pyproject.toml

```diff
@@ -20,13 +20,12 @@ license = "MIT"
 authors = [
   { name = "Chad Smith", email = "chadsmith.software@gmail.com" },
 ]
-requires-python = ">=3.8"
+requires-python = ">=3.9"
 classifiers = [
   "License :: OSI Approved :: MIT License",
   "Operating System :: OS Independent",
   "Programming Language :: Python",
   "Programming Language :: Python :: 3 :: Only",
-  "Programming Language :: Python :: 3.8",
```

Comment:
Could you also add `Programming Language :: Python :: 3.13` classifier at the bottom of the list?